mod_reforged/hooks/ambitions/ambition.nut

```diff
@@ -1,27 +1,41 @@
+::Reforged.HooksMod.hook("scripts/ambitions/ambition", function(q) {
+	q.m.ButtonIcon <- "ui/rf_default_ambition_icon.png";
+
+// New Functions
+	q.getButtonIcon <- function()
+	{
+		return this.m.ButtonIcon;
+	}
+});
+
 ::Reforged.QueueBucket.Late.push(function() {
 	::Reforged.HooksMod.hook("scripts/ambitions/ambition", function(q) {
 		q.getButtonTooltip = @(__original) function()
 		{
 			local ret = __original();
-			foreach (i, entry in ret)
+			if (this.getRenownOnSuccess() != 0)
 			{
-				if (("text" in entry) && entry.text == "Your Renown will increase, which means higher pay for contracts and potentially unlocking new types of contracts.")
+				local ret = __original();
```

Comment:
These changes to `getButtonTooltip` regarding Renown seem to have come into this PR by mistake. Please edit them out.